composer.json

```diff
@@ -11,6 +11,7 @@
     "require": {
         "php": "^7.3",
         "beberlei/assert": "^3.2",
+        "jwage/changelog-generator": "^0.0.7",
```

Comment:
Can we get a stabler release of this?

Reply:
I can work on the change to make it depend on psr18 instead of guzzle and release a 1.0.0

Reply:
Here we go https://github.com/jwage/changelog-generator/pull/50